<launch>
  <node name="republish_raw" pkg="image_transport" type="republish" output="screen" args="compressed in:=/orin16/mjpeg_cam/image raw out:=/orin16/mjpeg_cam/image"/>
</launch>